style.css

```diff
@@ -679,6 +696,146 @@ textarea {
   color: #fff;
 }
 
+.modal {
+  padding: 1rem;
+  display: none;
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  z-index: 99999;
+  background-color: #dededebb;
+  backdrop-filter: blur(0.3125rem);
+  -webkit-backdrop-filter: blur(0.3125rem);
+  -moz-backdrop-filter: blur(0.3125rem);
+}
+
+.modal-wrap {
+  padding: 1rem;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  justify-content: flex-start;
+  overflow: scroll;
+}
+
+.modal-header {
+  justify-content: space-between;
+  align-items: center;
+}
+
+.modal-title {
+  font-size: 2rem;
+  font-weight: 700;
+  line-height: 2.75rem;
+  color: #172b4d;
+  text-transform: capitalize;
+}
+
+.modal-content-wrapper {
+  gap: 0.75rem;
+}
+
+.modal-content-links {
+  justify-content: space-around;
+  align-items: center;
+  gap: 1.25rem;
+  width: 100%;
+}
+
+.modal-description {
+  line-height: 1.75rem;
+  width: 100%;
+  padding: 0.7rem;
+  text-align: start;
+}
```

Comment:
Great work team 🤝

- [ ] [Optional] Like the previous reviewer had mentioned, you may have to cut out most of that text. It is making your design very different from what is required
- [ ] [Optional] You may also need to reduce the size of your image on the desktop popup.
![content](https://user-images.githubusercontent.com/63917725/180601169-d27dcccd-dbb4-405a-a4ae-225f26d56dcf.PNG)